int32 xCenterUAV
int32 yCenterUAV
int32 xCenterImg
int32 yCenterImg
